### ISSUE TITLE
Is there anything special about your image of Loki or Promtail?

### ISSUE THREAD

**Issue by CarlQLange** [metalmatze/loki-kube-prometheus#1]
In your config, the only thing I see that worries me is that the Loki image is your own: `quay.io/metalmatze/loki:20190411`, and also promtail: `quay.io/metalmatze/promtail:20190411`. Can this be replaced with the standard official image, or is there something special about your fork?

Thanks for your work on this, it's surprising the kube-prometheus team haven't added Loki themselves.

**Comment by metalmatze** [comment]
Those images were copied during a time when there only was a loki `master` image available that was overwritten on every push. I wanted things to stabilize at least to some degree, that's why I created these snapshots. But they are exactly what `master` was at that time.

We should indeed update this repository to work with the latest images and contain the latest manifests.
In the end it would also make sense to see if upstream Loki ksonnet is independent from Consul and some other dependencies at that time.

**Comment by CarlQLange** [comment]
Ah, I see, very interesting. I didn't see anything related to Consul when researching this, maybe that was an older method.
Have you continued to use Loki with kube-prometheus? Any problems? Thanks a lot!

**Comment by metalmatze** [comment]
Yes, Consul and some other dependencies were in there when they released it way back in December. 
In the last few months my focus was mostly on [kube-thanos](https://github.com/thanos-io/kube-thanos) which recently moved upstream. But I am very happy to continue here and make sure it works with recent versions.

**Comment by CarlQLange** [comment]
Indeed, it looks like Consul is still used. It would be really great to have a way to use loki with kube-prometheus with no hassle!

**Comment by metalmatze** [comment]
Yes. That's exactly the reason why I started the project.
Happy to receive PRs, but maybe I can work on it a bit in the next days! 😊
